UI does not build

**Alluxio Version:**
2.2.0

**Describe the bug**
```
lerna ERR! npm run build --production  --no-save --no-package-lock --no-shrinkwrap exited 1 in '@alluxio/worker-ui'
lerna ERR! npm run build --production  --no-save --no-package-lock --no-shrinkwrap stdout:

> @alluxio/worker-ui@1.0.0 build /root/alluxio/webui/worker
> NODE_ENV=production && npm-run-all --parallel build-css build-js


> @alluxio/worker-ui@1.0.0 build-css /root/alluxio/webui/worker
> node-sass-chokidar --importer=../node_modules/node-sass-tilde-importer --include-path ./src --include-path ./node_modules src/ -o src/


> @alluxio/worker-ui@1.0.0 build-js /root/alluxio/webui/worker
> react-app-rewired build --scripts-version react-scripts

Rendering Complete, saving .css file...
Wrote CSS to /root/alluxio/webui/worker/src/containers/App/App.css
Rendering Complete, saving .css file...
Wrote CSS to /root/alluxio/webui/worker/src/index.css
Creating an optimized production build...
Failed to compile.

/root/alluxio/webui/worker/src/index.tsx
Type error: Property 'scroll' does not exist on type '{}'.  TS2339

    38 | 
    39 |   // Allow the client to control scroll-to-top using location.state
  > 40 |   if (loc.state && loc.state.scroll !== undefined && !loc.state.scroll) {
       |                              ^
    41 |     return;
    42 |   }
    43 |



lerna ERR! npm run build --production  --no-save --no-package-lock --no-shrinkwrap stderr:
Wrote 2 CSS files to /root/alluxio/webui/worker/src/
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @alluxio/worker-ui@1.0.0 build-js: `react-app-rewired build --scripts-version react-scripts`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @alluxio/worker-ui@1.0.0 build-js script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-04-02T07_25_33_290Z-debug.log
ERROR: "build-js" exited with 1.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @alluxio/worker-ui@1.0.0 build: `NODE_ENV=production && npm-run-all --parallel build-css build-js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @alluxio/worker-ui@1.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-04-02T07_25_33_306Z-debug.log
```

**To Reproduce**
mvn -T 2C clean package -DskipTests -Dmaven.javadoc.skip -Dfindbugs.skip -Dcheckstyle.skip -Dlicense.skip on FreeBSD 12.1 using node v10.17.0 and npm v6.12.1

**Expected behavior**
Compile!

**Urgency**
No real urgency

**Additional context**
Add any other context about the problem here.

--- a/webui/master/src/index.tsx
+++ b/webui/master/src/index.tsx
@@ -37,7 +37,8 @@ history.listen((loc: Location, action: Action) => {
   }
 
   // Allow the client to control scroll-to-top using location.state
-  if (loc.state && loc.state.scroll !== undefined && !loc.state.scroll) {
+  const l = (loc as unknown) as { state: { scroll: boolean } };
+  if (l.state && l.state.scroll !== undefined && !l.state.scroll) {
     return;
   }
 

--- a/webui/worker/src/index.tsx
+++ b/webui/worker/src/index.tsx
@@ -37,7 +37,8 @@ history.listen((loc: Location, action: Action) => {
   }
 
   // Allow the client to control scroll-to-top using location.state
-  if (loc.state && loc.state.scroll !== undefined && !loc.state.scroll) {
+  const l = (loc as unknown) as { state: { scroll: boolean } };
+  if (l.state && l.state.scroll !== undefined && !l.state.scroll) {
     return;
   }
 

